addons/survey/models/survey_survey.py

```diff
@@ -905,7 +905,7 @@ def action_result_survey(self):
         return {
             'type': 'ir.actions.act_url',
             'name': "Results of the Survey",
-            'target': 'self',
+            'target': '_blank',
```

Comment:
I was also under the impression that this is the correct way to do it but it looks like the "target" field on "ir.actions.act_url" values are:

```py
target = fields.Selection([('new', 'New Window'), ('self', 'This Window')],
                              string='Action Target', default='new', required=True)
```

I guess it still works since the task is greened but... could you take a look?
Looks like "new" is a lot more used for the target of actions when I grep the code.

Reply:
Indeed, I will change it to 'new' 
Should I change all the '_blank' occurence along with this change ? It seems they are all in the same file :smile:

Reply:
Sure why not, as long as it's in survey as well.
Make sure to include it in the commit message ("We took this opportunity to also clean blablabla").
Thanks!